x-pack/plugins/synthetics/common/constants/monitor_defaults.ts

```diff
@@ -70,21 +71,26 @@ export const DEFAULT_BROWSER_SIMPLE_FIELDS: BrowserSimpleFields = {
     is_zip_url_tls_enabled: false,
   },
   [ConfigKey.MONITOR_TYPE]: DataStream.BROWSER,
+  [ConfigKey.PARAMS]: '',
+  [ConfigKey.PORT]: undefined,
+  [ConfigKey.SCHEDULE]: {
+    unit: ScheduleUnit.MINUTES,
+    number: '10',
+  },
   [ConfigKey.SOURCE_INLINE]: '',
+  [ConfigKey.SOURCE_PUSH]: '',
   [ConfigKey.SOURCE_ZIP_URL]: '',
   [ConfigKey.SOURCE_ZIP_USERNAME]: '',
   [ConfigKey.SOURCE_ZIP_PASSWORD]: '',
   [ConfigKey.SOURCE_ZIP_FOLDER]: '',
   [ConfigKey.SOURCE_ZIP_PROXY_URL]: '',
-  [ConfigKey.PARAMS]: '',
   [ConfigKey.ZIP_URL_TLS_CERTIFICATE_AUTHORITIES]: undefined,
   [ConfigKey.ZIP_URL_TLS_CERTIFICATE]: undefined,
   [ConfigKey.ZIP_URL_TLS_KEY]: undefined,
   [ConfigKey.ZIP_URL_TLS_KEY_PASSPHRASE]: undefined,
   [ConfigKey.ZIP_URL_TLS_VERIFICATION_MODE]: undefined,
   [ConfigKey.ZIP_URL_TLS_VERSION]: undefined,
-  [ConfigKey.URLS]: undefined,
-  [ConfigKey.PORT]: undefined,
+  [ConfigKey.URLS]: '',
```

Comment:
just for the sanity purpose, we should probably sort these keys in alphabetical order